bin/varnishtest/tests/e00029.vtc

```diff
@@ -16,7 +16,7 @@ server s1 {
 	chunkedlen 0
 } -start
 
-varnish v1 -cliok "param.set thread_pool_stack 80k"
+varnish v1 -cliok "param.set thread_pool_stack 180k"
```

Comment:
this change defeats the purpose of the test.
Should we skip it for ASAN runs?

Reply:
I'm not sure if it is possible or overly specific but would it make sense to only skip the test if there is gcc and a sanitizer since it does pass fine on clang?

Reply:
Getting back to this, it looks like using GCC (11.4.0) with both asan and ubsan causes the failure. When using either asan or ubsan alone the test passes. This test seems like a much more nice to have with sanitizers on. Any thoughts? Just deal with the failure by saying it should be running with clang when using sanitizers?